BlogTemplate/Pages/Post.cshtml

```diff
@@ -27,6 +31,8 @@
                 <p name="Comment-Body" class="commentBody">@comment.AuthorEmail</p>
                 <p name="Comment-Date" class="commentDate">@comment.PubDate.ToString("MMM dd, yyyy")</p>
                 <p name="Comment-Body" class="commentBody">@comment.Body</p>
+                <a href="/post/@Model.Post.Slug" class="subtleLink">Reply</a>
```

Comment:
Do we have a story for replying to a comment?  Or is it just to leave a new comment?  This seems like a superfluous change right now.